addons/mail/static/src/discuss/core/common/message_model_patch.js

```diff
@@ -66,5 +66,12 @@ const messagePatch = {
             mentionedPartners,
         });
     },
+    /**
+     * @override
+     */
+    async remove() {
+        await super.remove();
+        this.parentMessage = false;
+    },
```

Comment:
Patching the `remove()` doesn't feel necessary, as the `RPC` is awaited and the data will be received from the bus anyway.

IIRC this was added to allow test coverage in tours that haven't the bus due to technical limitation. You do not seem to need it for a tour, and tours have bus notifications starting from 18.2 thanks to this PR: https://github.com/odoo/odoo/pull/189859